js/src/FormBuilder.d.ts

```diff
@@ -137,9 +141,13 @@ export declare class FormBuilder {
      * @param description Description of the payment shown in the form.
      * @param exitIframeOnResult Exit from iframe after a result. May be null.
      * @param shopLogoUrl The logo must be 250x250 pixel in .png format and must be hosted on a HTTPS (secure) server. Optional.
+     * @param phoneNumber User phone number with country code. Max AN 15. Optional.
+     * @param shopName Shop name. Max AN 100. Optional.
```

Comment:
Shop name displayed on MobilePay application. If omitted, the merchant name is used?